### DISCUSSION TITLE
Stuck at 176 messages?

### DISCUSSION THREAD

**Discussion by BigSweaterMan** [Tyrrrz/DiscordChatExporter#579] [Q&A] [answered]
Hi, I really wanted to download a conversation I had a while ago, but it seems that I can only download 176 messages. Changing the date doesn't change anything. Any advice?

**Comment by 96-LB** [comment, marked as answer]
Does this happen with every channel or just one? Is there anything unusual about the messages which fail to export?

**Reply by BigSweaterMan** [reply]
Just figured it out! Turns out I had multiple DMs open with the user. I had to archive three separate channels!

**Reply by Tyrrrz** [reply]
You mean multiple group channels? Normally I don't think you can have multiple 1-on-1 DM channels with the same user.
